rust/crd/src/lib.rs

```diff
@@ -363,6 +377,41 @@ impl AirflowRole {
 
         command
     }
+    fn authentication_start_commands(
+        auth_config: &AirflowClientAuthenticationDetailsResolved,
+    ) -> String {
```

Comment:
The commands are joined in this function, but the caller actually needs a vector:

```suggestion
    ) -> Vec<String> {
```

Reply:
done https://github.com/stackabletech/airflow-operator/pull/530/commits/b4d0e2fb818c5144a293d0583d4232eef9216661